Headers/CoreGraphics/CGContext.h

```diff
@@ -562,6 +578,19 @@ CGRect CGContextConvertRectToDeviceSpace(CGContextRef ctx, CGRect rect);
 
 CGRect CGContextConvertRectToUserSpace(CGContextRef ctx, CGRect rect);
 
+bool CGContextPathContainsPoint(CGContextRef c,
+    CGPoint point, CGPathDrawingMode mode);
+
+/* CoreGraphics Private APIs */
+
+bool CGContextGetShouldSmoothFonts(CGContextRef ctx);
+
+bool CGContextGetShouldAntialias(CGContextRef ctx);
+
+CGContextType CGContextGetType(CGContextRef ctx);
+
+void CGContextSetCTM(CGContextRef ctx, CGAffineTransform m);
```

Comment:
This is the one function I cannot find in the Apple header, this should be moved away.